common/sgx/edl/debug.edl

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** sgx/debug.edl:
+**
+**     Internal ECALLs/OCALLs to be used by liboehost/liboecore for operations
+**     related to debugging.
+**
+**     Note that this functionality is not inherently specific to SGX, it is
```

Comment:
Since this file is not inherently specific to SGX according to this line, should it be placed in a non-sgx-specific directory?